services/movie.py

```diff
@@ -0,0 +1,44 @@
+from typing import List
+from django.db.models.query import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: List[int] = None,
```

Comment:
```suggestion
        genres_ids: Optional[List[int]] = None,
```
Use Optional also because None as default

Reply:
Fix with all types